File: CareConnnect-client/careConnect-client/src/app/models/physicianResult.ts
export interface PhysicianResult {
  physicianId: number;
  userId: number;
  roleId: number;
  availability: boolean;
  specialty:string
  bio: string;
  fullName: string;
  email: string;
  phone: string;
  password: string | null;
  createdAt: Date;
}

File: CareConnect/Models/Database/results/PhysicianResult.cs
namespace CareConnect.Models.Database.results
{
    public class PhysicianResult
    {
        public int PhysicianId { get; set; }
        public int UserId { get; set; }
        public bool Availability { get; set; }
        public string Bio { get; set; }
        public string Specialty { get; set; }
        public string FullName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public int RoleID { get; set; } = 1;
        public DateTime? CreatedAt { get; set; }
    }
}
